frontend/components/create-review/review-form.tsx

```diff
@@ -1,11 +1,233 @@
-import React from 'react'
+import React, {useState} from 'react'
+import Button from '../ui/button'
+import ButtonLight from '../ui/button-light'
+import RatingsRadio from './ratings-radio'
+import postalCodes from 'postal-codes-js'
 
 //This components will hold the review form and it's data handling logic
 //Completed reviews should be sent to the backend with a success confirmation for the user (maybe need a Modal?)
 //Once completed, it should give an option to reset the form for another review or direct Client to Reviews page
 
+//TODO hook up with backend
+//TODO create error handling for regex tests
+
 function ReviewForm(): JSX.Element {
-	return <div></div>
+	const [landlord, setLandlord] = useState<string>('')
+	const [country, setCountry] = useState<string>('Canada')
+	const [city, setCity] = useState<string>('')
+	const [province, setProvince] = useState<string>('')
+	const [postal, setPostal] = useState<string>('')
+
+	const [repair, setRepair] = useState<number>(3)
+	const [health, setHealth] = useState<number>(3)
+	const [stability, setStability] = useState<number>(3)
+	const [privacy, setPrivacy] = useState<number>(3)
+	const [respect, setRespect] = useState<number>(3)
+
+	const [review, setReview] = useState<string>('')
+
+	const handleSubmit = (e: React.FormEvent) => {
+		e.preventDefault()
+		if (country === 'United States') {
+			if (!postalCodes.validate('us', postal)) {
+				//Set error message for Postal
+			}
+		} else {
+			if (!postalCodes.validate('CA', postal)) {
+				//set error message for Postal
+			}
+		}
```

Comment:
We should just use the country value here rather than requiring this logic. Line 108 - 109 should pull from the country json and use the country code as the value.